examples.py

```diff
@@ -1,28 +1,56 @@
+"""Refer to this file for example Python requests handled
+at the photonranch-projects API endpoints.
+
+All endpoints are handled at the base API URL
+"https://projects.photonranch.org/{stage}",
+here {stage} is the development stage in [test, dev, prod].
+
+Currently, both the production and development URLS point to dev.
```

Comment:
Not sure what you mean here--are you referring to the production and development frontends? If that's the case we probably don't need to include that info here as it is not super relevant, and it might become stale if the frontend changes to another scheme.

Reply:
Yup, I meant in the frontend, but I can remove it if irrelevant!